internal/configs/sharding/mainnet.go

```diff
@@ -9,9 +9,13 @@ import (
 )
 
 const (
-	mainnetEpochBlock1 = 344064 // 21 * 2^14
+	mainnetEpochBlock1 = 344064 // 21 * 2^14 (the last block of epoch 0)
 	blocksPerEpoch     = 16384  // 2^14
 
+	mainnetPrestakingEpochFirstBlock = 3336833 // TBD
```

Comment:
Need to change this block number after knowing the current beacon chain height in dryrun. @LeoHChen